mix.exs

```diff
@@ -38,16 +38,18 @@ defmodule VerkWeb.Mixfile do
       {:phoenix_pubsub, "~> 2.0"},
       {:phoenix_html, "~> 2.6"},
       {:gettext, "~> 0.16"},
-      {:verk, "~> 1.7.1"},
+      # {:verk, "~> 1.7.1"},
+      {:verk, github: "andreaswillems/verk", tag: "1.8.0"},
```

Comment:
Hey! Do we need a new Verk release as well?

Reply:
For testing and performance purposes, I replaced Jason with Jsonrs in Verk and somehow my change now appears in this PR. I close until I have figured out a clean way to contribute.